internal/api/v1/handlers/instance.go

```diff
@@ -48,10 +48,10 @@ func (h *InstanceHandler) ListInstances(c *fiber.Ctx) error {
 // CreateInstance handles the request to create a new instance
 func (h *InstanceHandler) CreateInstance(c *fiber.Ctx) error {
 	var req struct {
-		Name        string                           `json:"name"`
-		ProjectName string                           `json:"project_name"`
-		WebhookURL  string                           `json:"webhook_url"`
-		Instances   []infrastructure.InstanceRequest `json:"instances"`
+		InstanceName string                           `json:"instance_name"`
+		ProjectName  string                           `json:"project_name"`
+		WebhookURL   string                           `json:"webhook_url"`
+		Instances    []infrastructure.InstanceRequest `json:"instances"`
```

Comment:
This struct could have been defined as a request type instead of inline definition to improve reusability.

Reply:
I already resolved this somewhere... let me figure out which PR.

Reply:
ok I'm not crazy, I did it here: https://github.com/celestiaorg/talis/pull/77/files
Must have gotten lost in a merge conflict. 
I'll refactor again